src/Packetery/Module/CurrencySwitcherFacade.php

```diff
@@ -32,11 +31,16 @@ class CurrencySwitcherFacade {
 	 * @return float
 	 */
 	public function getConvertedPrice( float $price ): float {
-		if ( $this->isCurcyPluginEnabled() ) {
-			return (float) wmc_get_price( $price );
+		if ( Helper::isPluginActive( 'woocommerce-currency-switcher/index.php' ) ) {
```

Comment:
Proč je nutné zjišťovat, jestli je plugin WOOCS aktivní ? Proč prostě nemůžeme $price prohnat nejdříve filtrem WOOCS a pak filtrem packetery_price ? Tak jak je to teď se aplikuje buď filtr woocs nebo packeta_price.  Eshopista by měl mít možnost nechat přepočítat cenu filtrem woocs a pak na ni ve filtru packetery_price aplikovat nějaké další pravidla.  To že v example máme použití jiného currency switcheru je jen příklad použití a využití.

Reply:
Protože může být filtr definovaný někde jinde. Třeba ve functions.php. A já se chci vyhnout tomu, aby se cena zkonvertovala dvakrát. Chci to mít udělané tak, aby to bylo co nejhůře rozbitelné. Nedává mi smysl používat filtr cizího pluginu, když není aktivní.

Reply:
Aby se nemohla cena konvertovat dvakrát. Já chci aby packetery filtr byl jednoduchý a aby fungoval jako WooCS filtr, který se chová předvídatelně. Metoda getConvertedPrice() musí sloužit jen k převodu ceny z měny do měny. Navrhneš lepší název packetery filtru?

Reply:
OK. Necháme to zatím tak jak to je.